brain-marks/Localization/de.lproj/Localizable.strings

```diff
@@ -47,7 +48,7 @@
 "ReportABug" = "Melde einen Fehler";
 "GitHubRepo" = "GitHub Repository";
 "ThanksForUsing" = "Danke für die Nutzung von";
-"Version" = "Version";
+"Version " = "Version ";
```

Comment:
Why is there a space?

Reply:
I was thinking that there needed to be a space because so that the version number is not directly next to version like "Version1.1". However, I just realized that you have it in the view code with the space so it does not need to be in the Localized file. 

That is an incorrect change. Should I recommit without it?

Reply:
Yes! You can make the change and commit it to your branch, once you push, you'll see the PR is updated